README.md

```diff
@@ -408,6 +408,15 @@ render((
         uiSchema={uiSchema} />
 ), document.getElementById("app"));
 ```
+
+For fields for which the order is OK already or a fixed order is not important, you can insert a wildcard `"*"` item in your `ui:oder` definition. All unreferenced fields will be rendered at that point:
```

Comment:
Btw we may also want to add a short note about js/JSON object keys order being not consistently guaranteed, hence why explicitly defining the complete order is important if you care about UI consistency.